contracts/state/State.sol

```diff
@@ -70,7 +70,7 @@ contract State is Ownable2StepUpgradeable, IState {
         verifier = verifierContractAddr;
         _setDefaultIdType(defaultIdType);
         _gistData.initialize(MAX_SMT_DEPTH);
-        __Ownable_init();
+        __Ownable_init(msg.sender);
```

Comment:
looks like a change in State contract, version must be increased?

Reply:
I've rolled this change back cuz added two OpenZeppelin versions to the repo.